sct.py

```diff
@@ -619,7 +620,7 @@ def list_repos(dist_type, dist_version):
 @click.option('-r', '--scylla-repo', type=str,
               help='Scylla repo')
 @click.option('-d', '--linux-distro', type=str,
-              default='centos', help='Linux Distribution type')
+              default='ubuntu-focal', help='Linux Distribution type')
```

Comment:
what i'm afraid here is that we are adding this default in a 2nd place (we already have it in `test_default.yaml`), but if for now we don't initiate that default config file in time, this could be a workaround

Reply:
Yeah, it's more just to keep it in line with the current value in `test_default.yaml` if somebody runs the command manually. In the pipelines we provide the value from the calling function (even if it ends up being `null`)

Reply:
we could have no default now, as you are loading it from the defaults below...
so the verification could be:
```
if linux_distro or linux_distro == "null":
    linux_distro = test_defaults.get("scylla_linux_distro")
```

so we could remove the default for it, and it will inherit from the `test_default.yaml` if not provided from the pipeline